robottelo/ui/hostcollection.py

```diff
@@ -130,3 +130,72 @@ def add_host(self, name, host_name):
             (strategy, value % host_name), timeout=8)
         if element is None:
             raise UIError("Adding host {0} is failed".format(host_name))
+
+    def execute_bulk_package_action(self,
+                                    name,
+                                    action_name,
+                                    action_value=None,
+                                    action_via='katello_agent',
+                                    action_type='package',
```

Comment:
nitpick: i would place `action_type` before `action_value`, so the call would look like `execute_bulk_package_action(host, 'install', 'package', package_name)`, not `host, 'install', package_name, 'package')`, imo 'install package' sounds more naturally :)